analysers/analyser_osmosis_highway_vs_building.py

```diff
@@ -368,9 +368,9 @@ def __init__(self, config, logger = None):
         self.classs_change[5] = self.def_class(item = 1070, level = 2, tags = ['highway', 'waterway', 'geom', 'fix:imagery'], title = T_(u'Highway intersecting large water piece'), **doc)
         self.classs_change[6] = self.def_class(item = 1070, level = 2, tags = ['power', 'building', 'geom', 'fix:imagery'], title = T_(u'Power object intersecting building'), **doc)
         self.classs_change[7] = self.def_class(item = 1070, level = 2, tags = ['highway', 'power', 'geom', 'fix:imagery'], title = T_(u'Power object and highway too close'), **doc)
-        self.classs_change[8] = self.def_class(item = 1070, level = 2, tags = ['highway', 'geom', 'fix:imagery'], title = T_(u'Highway intersecting highway without junction'), **doc)
+        self.classs_change[8] = self.def_class(item = 1070, level = 2, tags = ['highway', 'geom', 'fix:imagery'], title = T_(u'Highway intersecting highway'), **doc)
         self.classs_change[9] = self.def_class(item = 1070, level = 2, tags = ['highway', 'geom', 'fix:imagery'], title = T_(u'Highway overlaps'), **doc)
-        self.classs_change[10] = self.def_class(item = 1070, level = 3, tags = ['waterway', 'geom', 'fix:imagery'], title = T_(u'Waterway intersecting waterway without junction'), **doc)
+        self.classs_change[10] = self.def_class(item = 1070, level = 3, tags = ['waterway', 'geom', 'fix:imagery'], title = T_(u'Waterway intersecting waterway'), **doc)
```

Comment:
So my logic for this one is that the "without junction" implies that there should be one when this isn't always the case (bridge, tunnel, etc.) which can be misleading to an uninformed user. If the title is for the issue itself, then surely the intersection is the issue?

Could add a `fix` string with suggestions to add a junction, tunnel, bridge, layer, etc.

Reply:
Perhaps "Waterway crossing waterway" is a bit clearer that the issue is with the ways crossing and there's no junction (since I suppose "intersecting" still applies when there is a junction).

Reply:
It is more about missing common node than missing layer on highway and even more on waterway.

Reply:
Maybe the "without junction" part could be moved to the detail string?